yellowme/youst-kotlin
# Feature/dark mode

## What does this PR do?
- Add new color palette
- Add dark-mode
- Add animation for dark-mode

## How should this be manually tested?
- By clicking this link in the phone https://yellowme.github.io/theme?wanted=dark the theme should change to dark. Close and open the app, the change should persist.
- By clicking this link in the phone https://yellowme.github.io/theme?wanted=light the theme should change to light. Close and open the app, the change should persist.

## What are the relevant tickets?
closes  #56
